src/appleseed/foundation/utility/string.h

```diff
@@ -404,6 +416,13 @@ inline std::string to_string(const uint8& value)
     return to_string(static_cast<unsigned int>(value));
 }
 
+// Handle nullptr object
+template <>
+inline std::string to_string(const std::nullptr_t& value)
```

Comment:
Oh, I had forgotten about `std::nullptr_t`. Do we actually need the `std::enable_if_t<>` logic then, can't we just provide an overload of `to_string()` for `std::nullptr_t`?

Reply:
This specialization is only available for `to_string(nullptr)`. It doesn't works if I make:
```cpp
void* p = nullptr; 
to_string(p);
```